aries-site/src/examples/templates/forms/SignUpExample.js

```diff
@@ -85,6 +85,8 @@ export const SignUpExample = () => {
     email: 'jane.smith@hpe.com',
   });
   const [passwordRules, setPasswordRules] = React.useState(passwordRulesStrong);
+  const [password] = React.useState('');
```

Comment:
With useState does the format need to be `[password, setPassword]` ?

Reply:
yeah I took out setPassword since I wasnt setting it I can put it back tho and just comment it out that its not being used

Reply:
I see, I think it would be helpful just to add a clarifying comment of why this is here. Something along the lines of this being used for demonstration purposes.

Reply:
done 👍